### PR TITLE
xen: rebase in-yocto patch onto the current HEAD

### PR DESCRIPTION
**Pull request by aanisov**
Signed-off-by: Andrii Anisov <andrii_anisov@epam.com>

### CODE REVIEW

**Comment by andr2000** [comment]
`Reviewed-by: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>`

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/xen/files/0001-ipmmu-vmsa-Disable-CONFIG_RCAR_IPMMU_PGT_IS_SHARED.patch [hunk 1]
```diff
@@ -11,18 +11,18 @@ Signed-off-by: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>
...
- 	depends on HAS_PASSTHROUGH && ARM_64
+ 	depends on HAS_PASSTHROUGH && ARM_64 && RCAR_IPMMU
 -	default y
 +	default n
  	help
  	  Enable this for Renesas R-Car's H3 v3.0 and M3N boards
  	  which have the IPMMU that does handle stage 2 translation
 -- 
-2.17.1
+2.7.4
```
Comment (andr2000) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/xen/files/0001-ipmmu-vmsa-Disable-CONFIG_RCAR_IPMMU_PGT_IS_SHARED.patch]: do you really need this?

**Comment by otyshchenko1** [comment]
lgtm

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/xen/files/0001-ipmmu-vmsa-Disable-CONFIG_RCAR_IPMMU_PGT_IS_SHARED.patch (hunk 1 above)
Comment (aanisov) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/xen/files/0001-ipmmu-vmsa-Disable-CONFIG_RCAR_IPMMU_PGT_IS_SHARED.patch]: You know, that patch is generated by git. I do not like manual modifications to the auto-generated file. So I tend to keep it as is.

**Comment by aanisov** [comment]
@iartemenko please merge

**Pull request merged**